src/views/StocksChartScreen.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, useState, useEffect } from "react";
```

Comment:
File name should be match your component name, if the folder name is `views` then file name should be `StockChart.view.tsx`. 
`StockChart` is not a correct name for page, A page can be `Home | Dashboard ` or something but the StockChart is component